packages/contracts/contracts/test/SecurityCouncil.t.sol

```diff
@@ -0,0 +1,183 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { SecurityCouncil_Initializer } from "./CommonTest.t.sol";
+
+contract SecurityCouncilTest is SecurityCouncil_Initializer {
+    /**
+     *  Events
+     */
+    event Confirmation(address indexed sender, uint256 indexed transactionId);
+    event Revocation(address indexed sender, uint256 indexed transactionId);
+    event Submission(uint256 indexed transactionId);
+    event Execution(uint256 indexed transactionId);
+    event ExecutionFailure(uint256 indexed transactionId);
+    event OwnerAddition(address indexed owner);
+    event OwnerRemoval(address indexed owner);
+    event RequirementChange(uint256 required);
+    event Validation(
+        uint256 indexed transactionId,
+        bytes32 outputRoot,
+        uint128 l2BlockNumber
+    );
+
+    struct Transaction {
+        address destination;
+        bool executed;
+        uint256 value;
+        bytes data;
+    }
+
+    struct OutputInfo {
+        bytes32 outputRoot;
+        uint128 l2BlockNumber;
+    }
+
+    function setUp() public virtual override {
+        super.setUp();
+    }
+
+    function test_initial_state() external {
+        address[] memory _owners = securityCouncil.getOwners();
+        for (uint256 i = 0; i < _owners.length; i++) {
+            assertEq(securityCouncil.owners(i), _owners[i]);
+        }
+
+        assertEq(securityCouncil.COLOSSEUM(), address(colosseum));
+        assertEq(securityCouncil.numConfirmationsRequired(), NUM_CONFIRMATIONS_REQUIRED);
+        assertEq(securityCouncil.getTransactionCount(true, true), 0);
+    }
+
+    function test_submit_transaction_reverts() external {
+        vm.prank(makeAddr("not colosseum"));
+        vm.expectRevert("SecurityCouncil: only the colosseum contract can be sender");
+        securityCouncil.submitTransaction(owners[0], 0, bytes("anydata"));
+    }
+
+    function test_submit_transaction_succeeds() external {
+        //submit dummy transaction
+        vm.prank(address(colosseum));
+        vm.expectEmit(true, false, false, false);
+        emit Submission(0);
+        uint256 txId = securityCouncil.submitTransaction(owners[0], 0, bytes("anydata"));
+
+        //check transaction count increased
+        Transaction memory t;
+        (t.destination, t.executed, t.value, t.data) = securityCouncil.transactions(txId);
+        assertEq(t.destination, owners[0]);
+        assertEq(t.executed, false);
+        assertEq(t.value, 0);
+        assertEq(t.data, bytes("anydata"));
+
+        uint256 txCount = securityCouncil.transactionCount();
+        assertEq(txCount, 1);
+    }
+
+    function test_confirm_transaction_reverts() external {
+        //submit dummy transaction
```

Comment:
```suggestion
        // submit dummy transaction
```

Reply:
Please put space after `//` to all the comments.

Reply:
Fixed as suggested.